.bumpversion.cfg

```diff
@@ -3,5 +3,8 @@ current_version = 0.5.1
 commit = True
 tag = False
 
-[bumpversion:file:docs/conf.py]
+[bumpversion:file:deploy/group_vars/amibuilder/main.yml]
+search = release: '{current_version}'
+replace = release: '{new_version}'
 
+[bumpversion:file:docs/conf.py]
```

Comment:
This bit tells `bumpversion` (the tool we use to consistently bump version numbers) that there's a version number in `deploy/group_vars/amibuilder/main.yml` that should be bumped.